pom.xml

```diff
@@ -96,7 +96,7 @@ under the License.
 		<slf4j.version>1.7.7</slf4j.version>
 		<guava.version>18.0</guava.version>
 		<akka.version>2.3.7</akka.version>
-		<java.version>1.7</java.version>
+		<java.minor.version>1.7</java.minor.version>
```

Comment:
Does this make a difference, or is that mainly a naming preference?
It it is only about naming, I would prefer to undo this change.
Call it "being conservative" with `pom.xml` changes - they have subtle implication son release scripts, etc.